assets/webpack.config.js

```diff
@@ -44,35 +44,14 @@ module.exports = (env, options) => ({
         test: /\.(png|woff|woff2|eot|ttf|svg)$/,
         loader: "file-loader",
         options: {outputPath: "css", publicPath: "/css"}
-      },
-      {
-        test: /bootstrap\/dist\/js\/umd\//,
-        loader: 'imports-loader',
-        options: {jQuery: 'jquery'}
       }
     ]
   },
   plugins: [
     new MiniCssExtractPlugin({ filename: 'css/app.css' }),
     new CopyWebpackPlugin({patterns: [{ from: 'static/', to: '.' }]}),
     new webpack.ProvidePlugin({
-      $: "jquery",
       jQuery: "jquery",
-      "window.jQuery": "jquery",
```

Comment:
As far as I understand, `ProvidePlugin` implicitly imports definitions from this dictionary to bundled js files. There's only one file in the application – `app.js`, and it doesn't use any of these definitions directly.

`app.js` uses some bootstrap functionality, like button tooltips and such, but they are available through `import "bootstrap"`.

I could be wrong, but I think they can be safely removed.